packages/chrome-extension/README.md

```diff
@@ -27,134 +27,56 @@
 - An existing React app (using [Vite](https://crxjs.dev/vite-plugin/) for example)
 
-### Installation
-
-1. Add `@clerk/chrome-extension` to your project:
-
-   ```shell
-   npm install @clerk/chrome-extension
-   ```
-
-1. Retrieve the **Publishable key** from your [Clerk dashboard](https://dashboard.clerk.com/last-active?path=api-keys) and set it as an environment variable. For example, if you used Vite:
-
-   ```sh
-   VITE_CLERK_PUBLISHABLE_KEY=pk_test_xxx
-   ```
-
-1. Add `<ClerkProvider>` to your app and define the `routerPush` & `routerReplace` properties. For example, with using `react-router-dom`:
-
-   ```tsx
-   // App.tsx
-   import { SignedIn, SignedOut, SignIn, SignUp, ClerkProvider } from '@clerk/chrome-extension';
-   import { useNavigate, Routes, Route, MemoryRouter } from 'react-router-dom';
-
-   function HelloUser() {
-     return <p>Hello user</p>;
-   }
-
-   const publishableKey = process.env.VITE_CLERK_PUBLISHABLE_KEY || '';
-
-   function ClerkProviderWithRoutes() {
-     const navigate = useNavigate();
-
-     return (
-       <ClerkProvider
-         publishableKey={publishableKey}
-         routerPush={to => navigate(to)}
-         routerReplace={to => navigate(to, { replace: true })}
-       >
-         <Routes>
-           <Route
-             path='/sign-up/*'
-             element={<SignUp signInUrl='/' />}
-           />
-           <Route
-             path='/'
-             element={
-               <>
-                 <SignedIn>
-                   <HelloUser />
-                 </SignedIn>
-                 <SignedOut>
-                   <SignIn
-                     afterSignInUrl='/'
-                     signUpUrl='/sign-up'
-                   />
-                 </SignedOut>
-               </>
-             }
-           />
-         </Routes>
-       </ClerkProvider>
-     );
-   }
-
-   function App() {
-     return (
-       <MemoryRouter>
-         <ClerkProviderWithRoutes />
-       </MemoryRouter>
-     );
-   }
-
-   export default App;
-   ```
-
-## Usage
-
-Example repositories:
+### Feature Support
 
-- [Standalone](https://github.com/clerk/clerk-chrome-extension-starter/tree/main): The extension is using its own authentication
-- [WebSSO](https://github.com/clerk/clerk-chrome-extension-starter/tree/webapp_sso): The extensions shares authentication with a website in the same browser
-
-### WebSSO
-
-If you want to use **WebSSO** (extension shares authentication state with a website in same browser) you'll need to add the `syncSessionWithTab` prop to `<ClerkProvider>`.
+Please see the latest extension [authentication support matrix](https://clerk.com/docs/references/chrome-extension/overview?utm_source=github&utm_medium=clerk_chrome_extension) in the Clerk documentation for more information.
 
-#### Extension Manifest (`manifest.json`)
+### Usage
```

Comment:
Split docs out into separate files as we've found that important details tended to be missed/buried otherwise.